src/pages/membership/components/createYouthProfileForm/CreateYouthProfileForm.tsx

```diff
@@ -409,7 +412,7 @@ function CreateYouthProfileForm(props: Props) {
                         id="pphotoUsageApprovedNo"
                         name="photoUsageApproved"
                         type="radio"
-                        value={false}
+                        value={'false'}
```

Comment:
Or this?

Reply:
Technically no. Photo usage radio button was broken, so I though I give it a quick fix. Now that I think of it, the fix is kinda unnecessary, because the value isn't even send to backend at the moment. I can make these changes at later date.